spec/api-power-monitor-spec.js

```diff
@@ -126,35 +128,32 @@ xdescribe('powerMonitor', () => {
     })
 
     describe('powerMonitor.querySystemIdleState', () => {
-      it('notify current system idle state', (done) => {
-        powerMonitor.querySystemIdleState(1, (idleState) => {
-          assert.ok(idleState)
+      it('notify current system idle state', done => {
+        powerMonitor.querySystemIdleState(1, idleState => {
+          expect(idleState).to.be.true()
           done()
```

Comment:
I've heard that async/await is cool =)

```js
it('notify current system idle state', async () => {
  const idleState = await new Promise(
      resolve => powerMonitor.querySystemIdleState(1, resolve))
  expect(idleState).to.be.true()
})
```

Reply:
It feels forced in this simple case, I would leave it as is

Reply:
Ok =/